### PR TITLE
SASS 920 - Added connectors for Delete Employment Financial Data API

### PR DESCRIPTION
**Pull request by ina-tu**
### Description
Added connectors and error scenarios for Delete Employment Financial Data API 

Add a link to the relevant story in Jira
[SASS-920](https://jira.tools.tax.service.gov.uk/browse/SASS-773)

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [x]  Have you checked to ensure all dependencies are up to date?

##### Whilst Reviewing
- [ ]  Have you run the tests?
- [ ]  Have you run the journey tests?
- [ ]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [ ]  Have you checked for merge conflicts?
- [ ]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you addressed warnings where appropriate?

##### After PRs been Reviewed
- [ ]  Have you checked the PR Builder passes?
- [ ]  Have you checked for merge conflicts?
- [ ]  Have you checked code coverage isn’t lower than previously?

### CODE REVIEW

File: test/services/EmploymentServiceSpec.scala [hunk 1]
```diff
@@ -92,11 +93,45 @@ class EmploymentServiceSpec extends TestUtils {
...
+
+        await(result) mustBe Right(())
+      }
+    }
+
+    "return Left containing DES Error" when {
+
+      "the deleteEmploymentFinancialData connector fails" in {
+
+        val desError = DesErrorModel(500, DesErrorBodyModel("DES_CODE", "DES_REASON"))
```
Comment (michael-rob-poole) [inline test/services/EmploymentServiceSpec.scala:123]: can be changed to INTERNAL_SERVER_ERROR to get rid of magic number warning

**Comment by michael-rob-poole** [comment]
image
can be updated

File: test/services/EmploymentServiceSpec.scala (hunk 1 above)
Comment (michael-rob-poole) [reply test/services/EmploymentServiceSpec.scala:123]: there is another line length warning in here as well.

Comment (ina-tu) [reply test/services/EmploymentServiceSpec.scala:123]: Sure

**Review by michael-rob-poole — APPROVED** [review]

**Pull request merged**